pom.xml

```diff
@@ -446,6 +446,33 @@
 					<autoVersionSubmodules>true</autoVersionSubmodules>
 				</configuration>
 			</plugin>
+      
```

Comment:
I see.  Is there another easy way for the javadocs and source to be included in projects that are importing XChange via maven and not directly including the checkout as a dependency in the project?